app/styles/_variables_mixins.scss

```diff
@@ -31,6 +32,7 @@ $sidebar_icon_lighter:  $navy_light;
 $sidebar_sublink_text:  #c2d2e3;
 $view_sub_nav:          #6784a2;
 $content_border:        $blue_lightest;
+$brown:                 #711214;
```

Comment:
![screenshot](https://cloud.githubusercontent.com/assets/16955978/23689963/17114bac-03e4-11e7-93af-d486a47092a9.png)

the $brown colour is used as font-colour of Cancel button on-hover state